### DISCUSSION TITLE
I get error while using drei Text element

### DISCUSSION THREAD

**Discussion by Achi00** [pmndrs/drei#865] [Q&A] [answered]
I get error while using drei Text element, I use it as I was using it before but it has problem with its anchorY and anchorX i removed it but still error any solutions? maybe it needs update or something?..
Thanks..!
![image](https://user-images.githubusercontent.com/101459158/163834258-83dd0ed2-410f-4526-8c96-a6dc98e09137.png)

**Comment by mwmwmw** [comment, marked as answer]
You will get this error if you attempt to use R3F components outside of the `<Canvas />` or inside an `<Html />` component

Check to make sure your components are in the right place. 

```javascript
// ✅ good
<Canvas>
<primitive object={myObj} />
</Canvas>

// ❌bad
<Canvas>
 /* etc ... */
</Canvas>
<primitive object={myObj} />

// ❌bad
<Canvas>
  <Html>
    <primitive />
  </Html>
</Canvas>

```
